qiskit/visualization/pass_manager_visualization.py

```diff
@@ -98,19 +99,27 @@ def pass_manager_drawer(pass_manager, filename=None, style=None, raw=False):
 
     HAS_GRAPHVIZ = has_graphviz  # pylint: disable=invalid-name
 
+    if not HAS_GRAPHVIZ:
```

Comment:
It looks like we're checking twice for graphviz, is this intended?

Reply:
not intended indeed! One of the checks is being removed in 4b46f1450